appsec/src/extension/php_compat.c

```diff
@@ -6,6 +6,13 @@
 #include "php_compat.h"
 
 #if PHP_VERSION_ID < 70300
+static zend_string _zend_empty_string_st = {
+    .gc.refcount = 1,
```

Comment:
Are you sure it's fine? IIRC interned strings need at least RC=2. But maybe on 7.2 and prior RC=1 was fine too.